pillar.example

```diff
@@ -14,12 +15,12 @@ golang:
     altpriority: 9100
 
   cmd:
-    goget:
-      - github.com/golang/example/hello
-      - github.com/golang/example/outyet
-    clean:
-      - github.com/golang/example/hello
-      - github.com/golang/example/outyet
+    goget: {}
+    # github.com/golang/example/hello
+    # github.com/golang/example/outyet
+    clean: {}
+    # github.com/golang/example/hello
+    # github.com/golang/example/outyet
```

Comment:
Since this is used as a list:

https://github.com/saltstack-formulas/golang-formula/blob/32e1fc77d4aa351a49f01b335402c665280828f9/golang/cmd/goget.sls#L7

This appears to be more appropriate:

```suggestion
    goget: []
    #   - github.com/golang/example/hello
    #   - github.com/golang/example/outyet
    clean: []
    #   - github.com/golang/example/hello
    #   - github.com/golang/example/outyet
```

* Specifically, using empty lists (`[]`) instead of empty dicts (`{}`).